src/Microsoft.Azure.WebJobs.Host/Scale/IScaleStatusProvider.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Interface for providing a scale status
```

Comment:
"Interface for providing aggregate scale status across all functions being monitored by the host."